src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,210 @@
+.search-filter-container {
+  display: flex;
```

Comment:
yes i need this diplay flex because the array title, datePosted, and minimumEvaluation when are maping they comes as a list with flex-direction:column